src/AbstractDifferentiation.jl

```diff
@@ -43,20 +47,19 @@ function gradient(ab::AbstractBackend, f, xs...)
     return reshape.(adjoint.(jacobian(lowest(ab), f, xs...)),size.(xs))
 end
 function jacobian(ab::AbstractBackend, f, xs...) end
+function jacobian(ab::HigherOrderBackend, f, xs...) 
+    jacobian(lowest(ab), f, xs...)
+end
+
 function hessian(ab::AbstractBackend, f, xs...)
```

Comment:
Should we change `xs...` to `xs::AbstractVector`?

Reply:
or just `x`

Reply:
sounds good. There is one tiny downside: 
if `x` is not a tuple, we need to write:
`value = primalvalue(ab, v, f, (x,))`
or define additional dispatches for `primalvalue`.